pydm/widgets/timeplot.py

```diff
@@ -1151,6 +1204,38 @@ def enableCrosshair(
             horizontal_movable,
         )
 
+    def updateLabel(self, x_val: float, y_val: float) -> None:
+        """
+        Update the label for each curve with severity information, if available.
+
+        This method first calls the parent class's `updateLabel` to handle general
+        label updates and positioning. It then appends severity information to the
+        label of each curve that contains a valid `severity_raw` attribute.
+
```

Comment:
I feel like for the timeplot/archiver plot labels, running the x value through `time.strftime` or some such would be helpful? Having human-readable dates/times rather than timestamps for x values seems more useful.

Reply:
oh, yes that's a good idea. i actually have a method for it in the code, getFormattedX,  just forgot to use it  ;)

Reply:
ok, made the fix and added a getFormattedY as well